### PR TITLE
Changed consensus to manual-seal

### PR DESCRIPTION
**Pull request by PierreOssun**
Hello,
I've changed consensus from `aura` & `grandpa` with 6 seconds block-time.
To `manual-seal` where blocks are authored at every transaction.
It makes transactions almost instant and is very practical for hacking & testing

### CODE REVIEW

**Review by athei — APPROVED** [review]
Nice. Thanks for that. This makes development much more convenient.

**Comment by xgreenx** [comment]
It closes https://github.com/paritytech/substrate-contracts-node/issues/36

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -235,46 +211,27 @@ impl frame_system::Config for Runtime {
...
-	type HandleEquivocation = ();
-
-	type WeightInfo = ();
-	type MaxAuthorities = MaxAuthorities;
-}
+impl pallet_randomness_collective_flip::Config for Runtime {}
 
 parameter_types! {
-	pub const MinimumPeriod: u64 = SLOT_DURATION / 2;
+	pub const MinimumPeriod: u64 = 5;
```
Comment (HCastano) [inline runtime/src/lib.rs:228]: Not sure we really need a minimum period here. Do you know if we can simply set this to 0?

**Review by HCastano — COMMENTED** [review]
Just one small question about the `MinimumPeriod`.

Tried it locally and it works great! What resource did you end up using to implement this?

File: runtime/src/lib.rs (hunk 1 above)
Comment (PierreOssun) [reply runtime/src/lib.rs:228]: MinimumPeriod is The minimum period between blocks production in milliseconds. 
But it should not be set as 0 as it is used by set function (set time) in pallet timestamp and invoked once per block and will prevent the block from finalization.
[see pallet-timestamp](https://github.com/paritytech/substrate/blob/96738a1f3dbe4a1fb6abb385920fb403ee961488/frame/timestamp/src/lib.rs#L186)
As for setting it to 1 made zero difference on my test so I would keep 5 as a minimum secure threshold

Comment (HCastano) [reply runtime/src/lib.rs:228]: Ah, I didn't read the docs carefully enough - I just assumed it was in seconds. Five milliseconds sounds totally reasonable to me

**Comment by HCastano** [comment]
> @HCastano I've discussed with @atenjin from patract labs who has already changed consensus to manual-seal in their `jupiter` & `europa` nodes. And look at the work done on these nodes

Would be cool to see a short blog post about this 😉

**Review by HCastano — APPROVED** [review]
Thanks!

**Pull request merged**